mlir-sycl/include/mlir/Dialect/SYCL/Transforms/Passes.td

```diff
@@ -11,6 +11,26 @@
 
 include "mlir/Pass/PassBase.td"
 
+def ConstantPropagationPass : Pass<"sycl-constant-propagation", "gpu::GPUFuncOp"> {
+  let summary = "SYCL host-device constant propagation";
+  let description = [{
+    This pass implements SYCL host-device constant propagation. In order to do
+    so, it tracks constant values, e.g., those defined by `arith.constant`,
+    passed as an argument to `sycl.host.schedule_kernel`, and replaces the
+    corresponding `gpu.func` argument with the constant value.
+
+    No further change is needed to the `sycl.host.schedule_kernel` operation or
+    the kernel, as the `DAESYCL` pass will remove the kernel arguments that
```

Comment:
Can we spell out `DAESYCL` (dead argument elimination for SYCL) here once and add that this is an LLVM pass, not an MLIR pass?